api/base/serializers.py

```diff
@@ -983,18 +997,7 @@ def to_representation(self, value):
         """
         meta = functional.rapply(self.meta, _url_val, obj=value, serializer=self.parent, request=self.context['request'])
         obj = getattr(value, 'referent', value)
-        return {
```

Comment:
Add back in - removed as a result of the rebase after schemablocks was merged into develop

Reply:
🆗